src/components/projects_home/allExpenses.tsx

```diff
@@ -54,9 +52,11 @@ function AllExpenses(props: any) {
         >
           Expenses
           <span className="material-icons ml-4 mr-2 icon">attach_money</span>
-        </h2>
-        <h2 style={{ textAlign: "center" }}>${total}</h2>
-
+        </h2> */}
+        {/* <h2 style={{ textAlign: "center" }}>${total}</h2> */}
+        <h3 style={{ textAlign: "center" }}>
+          {expenses.length <= 0 ? "No expenses have been created yet" : ""}
```

Comment:
So that when the user has no expenses created he knows that's why the page is blank